README.md

```diff
@@ -1,49 +1,103 @@
 # migrator
 
-Command line interface (CLI) to migrate a computer to run balenaOS.
+Command line interface (CLI) to automate migration of a computer to run balenaOS.
 
 **WARNING This tool will overwrite the operating system and all data on the computer that runs it.** Be sure to save any important data before using the migrator.
 
+Presently supports migration only from a computer running Windows 10 on UEFI based firmware, with Ethernet DHCP (self-configured) networking.
+
 ## About
 
-The migrator provides a command prompt / terminal window to run etcher SDK's [migrator function](https://github.com/balena-io-modules/etcher-sdk/tree/master/lib/migrator). The migrator is an [open-source project on
-GitHub](https://github.com/balena-os/migrator/), and your contributions are welcome!
+Migrator accepts a balenaOS image file for your computer, formats it onto  the computer's disk/flash storage, and then replaces the current operating system with balenaOS derived from that image. As a CLI application, it's easy to automate, and you can run it remotely.
 
-Presently supports migration only from a computer running Windows 10 on UEFI based firmware.
+*Our goal is to make it easy to migrate a computer to use balenaOS.* Let us know how it works for you. As an open source project, we're also happy to accept code and documentation contributions. See *Development* below.
 
-## Installation and Building
+## Getting Started
 
-Clone the migrator repository, install the node modules and build it as shown below.
+### Prepare image
+Migrator expects a pre-configured image for your device. You only need to perform these download and configure steps once for a fleet.
 
+Use [balenaCLI](https://docs.balena.io/reference/balena-cli/#os-download-type) or the balena [web site](https://www.balena.io/os#download-os) to download the image appropriate for your computer. Then configure it for your fleet. If you download from the web site, first unzip the compressed file.
 ```
-> git clone https://github.com/balena-os/migrator.git
-> cd migrator
-> npm install
-> npm run build
-> npm run pkg
+  > balena os download generic-amd64 -o balena-flasher.img
+     Getting device operating system for generic-amd64
+     OS version not specified: using latest released version
+     balenaOS image version 2.113.12 downloaded successfully
+
+  > balena os configure balena-flasher.img --fleet MyFleet --version 2.113.12
```

Comment:
I would also mention that they should make sure balenaOS can run in their device and it appears in the dashboard. Maybe link to a Getting Started for Generic AMD64 as an example.

Also, a note that the device needs to have an ethernet and an DHCP server needs to be configured in the network is an important caveat until configuration migration is supported.

Reply:
@kb2ma echoing my opinion above, I don't think this README should delve into device specifics. That information will just be unmaintained and outdated. If needed I would modify specific device repositories to say that migration is supported.

Reply:
I assume the only supported flasher images ATM are the two generic x86, which answers the question.